spacy/lang/tn/__init__.py

```diff
@@ -0,0 +1,18 @@
+from .stop_words import STOP_WORDS
+from .lex_attrs import LEX_ATTRS
+from .punctuation import TOKENIZER_INFIXES
+from ...language import Language
+
+
+class SetswanaDefaults(Language.Defaults):
+    suffixes = TOKENIZER_SUFFIXES
```

Comment:
I think you want to do this instead?
```suggestion
    infixes = TOKENIZER_INFIXES
```